### PR TITLE
(tablet/desktop) gallery

### LINKED ISSUE #107: [Frontend] Gallery (Tablet, Desktop)
## 🌟 User Story

**As a** tablet / desktop user,  
**I want** be able to view the gallery,  Ï

---

### 🎯 **Acceptance Criteria**

- [ ] The gallery should adhere to the design
- [ ] There should be arrows for me to click / it should be horizontally scrollable
- [ ] Edge cases -- what if there are 
  - [ ] 0 gallery images
  - [ ] 1 gallery image
  - [ ] 2 gallery images

---

### 📊 **Definition of Done (DoD)**

- [ ] Acceptance criteria met
- [ ] Agreed tests written
- [ ] Agreed documentation in GitHub wiki
- [ ] PR reviewed

---

### 🔗 **Dependencies**

---

### PR DESCRIPTION
**Pull request by Troyabbs**
closes #107 

## 🌟 Context 
Mobile/Desktop view Gallery

---

## 📝 Description 

Opacity on edge images,
Button placement changed,
Title placement changed,

---

### CODE REVIEW

File: src/app/(frontend)/_components/home/gallery/ImageSlider.tsx [hunk 1]
```diff
@@ -36,6 +50,12 @@ export default function GallerySlider({ gallery }: Props) {
         slider.on('created', nextTimeout)
         slider.on('dragStarted', () => clearTimeout(timeout))
         slider.on('animationEnded', nextTimeout)
+        slider.on('detailsChanged', (s) => {
+          s.track.details.slides.forEach((slide, index) => {
+            const individualSlide = s.slides[index]
+            individualSlide.style.opacity = slide.portion > 0.99 ? '1' : '0.5'
```
Comment (dyzhuu) [inline src/app/(frontend)/_components/home/gallery/ImageSlider.tsx:56]: opacity on design is 0.6

**Review by dyzhuu — CHANGES_REQUESTED** [review]
Just my other comments during the meeting
- using `cursor-grab`
- make it horizontally scrollable
- maybe adding clickable arrows as well

**Review by Troyabbs — COMMENTED** [review]
done

**Review by dyzhuu — APPROVED** [review]
LGTM, great work - love the parallax!

**Review by BlackPri0r — APPROVED** [review]
LGTM - this guy kinda cracked no?
